packages/select/src/MenuButton.tsx

```diff
@@ -82,7 +141,7 @@ const menuButtonTextStyle = css`
 
 const errorColor: Record<Mode, string> = {
   [Mode.Light]: palette.red.base,
-  [Mode.Dark]: '#F97216',
+  [Mode.Dark]: '#FF6960', // TODO: palette.red.light1 is different in figma(#FF6960) which does not match react(#EF5752)
```

Comment:
Ah gotcha. It might be good to note on that ticket that these `todo` comments should be resolved together with that change - might get lost otherwise.